cloud-minecraft/cloud-brigadier/src/main/java/cloud/commandframework/brigadier/CloudBrigadierManager.java

```diff
@@ -574,6 +593,12 @@ public void registerDefaultArgumentTypeSupplier(
         }
 
         for (final String suggestion : suggestions) {
+            // Skip suggestions that are sibling literals
+            // They are already part of the literal argument construction earlier
+            if (siblingLiterals.contains(suggestion)) {
```

Comment:
This can be done before the loop using `suggestions.removeIf(siblingLiterals::contains)`. While this would require an extra loop, I would prefer keeping that logic outside of the "rendering" loop

Reply:
I don't think the list returned by suggest() is modifiable like that, but I could stream(), filter() and collect() it into a new List